File: C++ for Everyone, 2nd Ed, Horstmann/Chapter3/P3-4.cpp
//Write a program that reads in three strings and sorts them lexicographically.
//Enter three strings : Charlie Able Baker
//Able
//Baker
//Charlie

#include<iostream>

using namespace std;

int main()
{
    string s1, s2, s3;

    cout << "Enter three strings: ";
    cin >> s1 >> s2 >> s3;

    if(s1 < s2 && s1 < s3)
    {
        cout << s1 << "\n";

        if(s2 < s3)
        {
            cout << s2 << "\n";
            cout << s3 << "\n";
        }
        else
        {
            cout << s3 << "\n";
            cout << s2 << "\n";
        }
    }

    else if(s1 > s2 && s2 < s3)
    {
        cout << s2 << "\n";

        if(s1 < s3)
        {
            cout << s1 << "\n";
            cout << s3 << "\n";
        }

        else
        {
            cout << s3 << "\n";
            cout << s1 << "\n";
        }
    }

    else
    {
        cout << s3 << "\n";

        if(s1 < s2)
        {
            cout << s1 << "\n";
            cout << s2 << "\n";
        }

        else
        {
            cout << s2 << "\n";
            cout << s1 << "\n";
        }
    }

    return 0;
}

File: Python-For-Everyone-Horstmann/Chapter3-Decisions/P3.4.py
# Write a program that reads three numbers and prints “all the same” if they are all the
# same, “all different” if they are all different, and “neither” otherwise.

number1 = float(input("Enter number one: "))
number2 = float(input("Enter number two: "))
number3 = float(input("Enter number three: "))

if number1 == number2 and number2 == number3 :
    print("They're all the same")

elif number1 != number2 and number2 != number3 and number1 != number3:
    print("They're all different")

else:
    print("Neither")